fml/backtrace.cc

```diff
@@ -4,27 +4,32 @@
 
 #include "flutter/fml/backtrace.h"
 
-#include <cxxabi.h>
-#include <dlfcn.h>
-#include <execinfo.h>
-
 #include <csignal>
 #include <sstream>
 
-#if FML_OS_WIN
-#include <crtdbg.h>
-#include <debugapi.h>
-#endif
-
+#include "flutter/fml/build_config.h"
 #include "flutter/fml/logging.h"
 #include "flutter/fml/paths.h"
 #include "third_party/abseil-cpp/absl/debugging/symbolize.h"
 
+#ifdef FML_OS_WIN
+#include <Windows.h>
+#include <crtdbg.h>
+#include <debugapi.h>
+#else  // FML_OS_WIN
+#include <cxxabi.h>
+#include <dlfcn.h>
+#include <execinfo.h>
+#endif  // FML_OS_WIN
+
 namespace fml {
 
 static std::string kKUnknownFrameName = "Unknown";
 
 static std::string DemangleSymbolName(const std::string& mangled) {
+#if FML_OS_WIN
```

Comment:
@chinmaygarde for thoughts on preprocessor conditionals vs making a posix subclass and a windows subclass? That's something we do with things like files, mutexes etc. typically by adding a platform specific file in [`fml/platform`](https://github.com/flutter/engine/tree/main/fml/platform/) that's conditionally compiled in via the BUILD file. e.g. For files we do:
* https://github.com/flutter/engine/blob/main/fml/file.h
* https://github.com/flutter/engine/blob/main/fml/platform/posix/file_posix.cc